test/utilities/translator.js

```diff
@@ -136,6 +137,7 @@ export function createFindTermsOptions(dictionaryName, optionsPresets, optionsAr
         sortFrequencyDictionary: typeof sortFrequencyDictionary !== 'undefined' ? sortFrequencyDictionary : null,
         sortFrequencyDictionaryOrder: typeof sortFrequencyDictionaryOrder !== 'undefined' ? sortFrequencyDictionaryOrder : 'ascending',
         removeNonJapaneseCharacters: typeof removeNonJapaneseCharacters !== 'undefined' ? removeNonJapaneseCharacters : false,
+        primaryReading: typeof primaryReading !== 'undefined' ? primaryReading : '',
```

Comment:
would this work the same? wondering why the ones above it  explicitly check the type
```suggestion
        primaryReading: primaryReading ?? '',
```

Reply:
`??` also checks for null, should work the same but I don't think it is needed since `primaryReading` is `string | undefined`

Reply:
actually wouldn't this mean that it can be returned as null if it was null to start with? when we'd probably want it to be a string in 100% of cases.

Reply:
not a big deal but i really wonder why nutbread decided to do it this way for the ones above it.

Reply:
This is the typing for test inputs. Either the test `option` does not specify the `primaryReading` or has `primaryReading` as a string. It cannot be null unless we explicitly put null for `primaryReading` here (which is not a valid test case).

```
        {
            "name": "Find terms using primary reading 1",
            "func": "findTerms",
            "mode": "split",
            "text": "自重",
            "options": [
                "default",
                {
                    "type": "terms",
                    "primaryReading": "じちょう"
                }
            ]
        }
 ```

Reply:
> not a big deal but i really wonder why nutbread decided to do it this way for the ones above it.

some props have explicit null value so probably that's the reason
```
        sortFrequencyDictionary: typeof sortFrequencyDictionary !== 'undefined' ? sortFrequencyDictionary : null,
```